eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -243,11 +280,10 @@ jobs:
               ${{ insert }}: ${{ matrixEntry.value }}
 
     pool:
-      vmImage: $(OSVmImage)
+      name: $[coalesce(variables['Pool'], '')]
```

Comment:
Is the coalesce needed? Won't this return empty if the variable isn't set?

Reply:
No ... the value will just stay as $(OSVmImage) instead of evaluating to an empty string, so this overcomes that.

Reply:
Ok I expected that if you did `$[variables['Pool']]` that the right thing would happen.